example/webpack1-babel/webpack.conf.js

```diff
@@ -18,25 +18,17 @@ module.exports = {
     main: './test/main_test',
   },
 
-  plugins: [
-    new webpack.DefinePlugin({ __VALUEA__: 10 }),
-  ],
+  plugins: [new webpack.DefinePlugin({__VALUEA__: 10})],
 
   output: {
     path: path.resolve(__dirname, './dest'),
-    filename: 'bundle.js',
+    filename: '[name].js',
```

Comment:
this is the fix for the aforementioned `karma-webpack` issue. In 3.0.0 (which was the latest version the last time `inject-loader` was released), this filename was always hardcoded to `[name]`, but then that changed with https://github.com/webpack-contrib/karma-webpack/pull/336, which also introduced the error we're seeing now. Unfortunately, the fix never made it into a 3.x release so we could either pin to karma-webpack 3.0.0 or make the change I made here.